languages/python/bitwarden_sdk/bitwarden_client.py

```diff
@@ -62,13 +93,18 @@ def list(self, organization_id: str) -> ResponseForSecretIdentifiersResponse:
         )
         return ResponseForSecretIdentifiersResponse.from_dict(result)
 
-    def update(self, id: str,
-               key: str,
-               note: str,
-               organization_id: str,
-               value: str,
-               project_ids: Optional[List[UUID]] = None
-               ) -> ResponseForSecretResponse:
+    def update(
+            self,
+            organization_id: str,
+            id: str,
+            key: str,
+            value: str,
+            note: str,
+            project_ids: Optional[List[UUID]] = None,
+    ) -> ResponseForSecretResponse:
+        if note is None:
```

Comment:
Since `note` can be None, it should be `Optional[str]`

Reply:
Fixed in 86bb6de6d02ac1835cd4e511615b50fe36975fda.